### PR TITLE
Implement github and linkedin links on profile

### PR DESCRIPTION
**Pull request by jonasdeluna**
Implemented github and linkedin fields on user.

Blocking: https://github.com/webkom/lego-webapp/pull/3901

### CODE REVIEW

File: lego/apps/users/constants.py [hunk 1]
```diff
@@ -80,10 +80,10 @@ class FSGroup(Enum):
...
         return [e.value for e in cls]
 
-    MTDT = "fc:fs:fs:prg:ntnu.no:mtdt"
-    MTKOM = "fc:fs:fs:prg:ntnu.no:mtkom"
-    MIDT = "fc:fs:fs:prg:ntnu.no:midt"
-    MSTCNNS = "fc:fs:fs:prg:ntnu.no:mstcnns"
+    MTDT = "fc:fs:fs:prg:ntnu.no:MTDT"
+    MTKOM = "fc:fs:fs:prg:ntnu.no:MTKOM"
+    MIDT = "fc:fs:fs:prg:ntnu.no:MIDT"
+    MSTCNNS = "fc:fs:fs:prg:ntnu.no:MSTCNNS"
```
[the remark this answers is not in the document]
Comment (haukkagu) [reply lego/apps/users/constants.py:86]: It may be because of a rebase? But they shouldn't be apart of the PR right?

**Review by haukkagu — APPROVED** [review]
LGTM

File: lego/apps/users/constants.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonasdeluna) [reply lego/apps/users/constants.py]: Thanks for the heads up, yeah it was. Should be fixed now!

[the remark this answers is not in the document]
Comment (haukkagu) [reply lego/apps/users/constants.py]: 🔥🔥🔥

**Pull request merged**